Source/Meadow.Foundation.Libraries_and_Frameworks/Graphics.MicroLayout/Driver/Charts/LineChart.cs

```diff
@@ -194,31 +194,30 @@ private void DrawXAxis(MicroGraphics graphics, double minY, double maxY)
             // axis is at 0
             XAxisYIntercept = 0;
 
-            XAxisScaledPosition = Bottom - DefaultMargin - DefaultAxisStroke + (int)(minY * VerticalScale);
+            XAxisScaledPosition = Bottom - DefaultMargin + (int)(minY * VerticalScale);
         }
         else
         {
             // axis at min Y
             XAxisYIntercept = YMinimumValue;
 
-            XAxisScaledPosition = Bottom - DefaultMargin - DefaultAxisStroke;
+            XAxisScaledPosition = ChartAreaBottom - DefaultMargin * 2 + AxisStroke * 2;
         }
 
         // for now it's a fixed line at the bottom
-        graphics.Stroke = DefaultAxisStroke;
-        graphics.DrawLine(
-            ChartAreaLeft + ParentOffsetX,
-            XAxisScaledPosition + ParentOffsetY,
-            Right - DefaultMargin,
+        graphics.Stroke = AxisStroke;
+        graphics.DrawHorizontalLine(
+            ChartAreaLeft, //ChartAreaLeft - AxisStroke + ParentOffsetX,
```

Comment:
Is the ParentOffsetX not needed?

Reply:
@adrianstevens I wonder that same thing. I presumed is when drawing Linecharts in the middle of the screen, but testing that as is, I found another bug, since the control is not rendered correctly at all:

![image](https://github.com/WildernessLabs/Meadow.Foundation/assets/3408668/4ecd5ed7-d5e2-4252-aa3b-835523781594)

My PR fixes that scenario as well:

![image](https://github.com/WildernessLabs/Meadow.Foundation/assets/3408668/419a39c6-5105-48b7-916d-ff0308fb5ed2)